v0/destinations/posthog/data/PHAliasConfig.json

```diff
@@ -0,0 +1,17 @@
+[
+  { "destKey": "properties.distinct_id", 
+    "sourceKeys": [
+    "userId",
+    "context.traits.userId",
+    "context.traits.id",
+    "anonymousId"
+  ], 
+  "required": true },
+  { "destKey": "properties.alias", "sourceKeys": "previousId", "required": true },
+  {
+      "destKey": "timestamp",
+      "sourceKeys": ["originalTimestamp", "timestamp"],
+      "required": true
+  },
+  { "destKey": "context", "sourceKeys": "context", "required": true }
```

Comment:
we cleared our confusion around context ?
it's better not to send if not

Reply:
Fixed